src/All/README.md

```diff
@@ -66,6 +74,10 @@ All(false).concat(All.empty())  //=> All false
 ```haskell
 All ~> All -> All
 ```
+<details>
+  <summary>Breakdown</summary>
+  <code> All.concat </code> inputs type <code> All </code> and outputs type <code> All </code>
```

Comment:
I am thinking instead of providing text blocks that the user needs to parse through, we could come up with a 🌽sistant  list format that lists each in order by arrow, with the last always being a return. so this could be:

```html
<dl>
  <dt>all<dt><dd>The <code>All</code> to combine<dd>
  <dt><i>Result</i><dt><dd>The combined <code>All</All><dd>
</dl>
```